tools/integration-test/src/tests/consensus_states.rs

```diff
@@ -37,14 +38,14 @@ impl BinaryChainTest for ConsensusStateHeights {
             }
         }
 
-        let states = chains
-            .handle_b()
-            .query_consensus_states(QueryConsensusStatesRequest {
-                client_id: (*chains.client_id_b().value()).clone(),
-                pagination: Some(PageRequest::all()),
-            })?;
+        // let states = chains
+        //     .handle_b()
+        //     .query_consensus_states(QueryConsensusStatesRequest {
+        //         client_id: (*chains.client_id_b().value()).clone(),
+        //         pagination: Some(PageRequest::all()),
+        //     })?;
 
-        assert_eq!(states.len(), CONSENSUS_STATES_COUNT);
+        // assert_eq!(states.len(), CONSENSUS_STATES_COUNT);
```

Comment:
remove commented out code, also below.

Reply:
I updated the test: https://github.com/informalsystems/hermes/pull/2990/commits/a45b6e99ac48488b929b627bfb09135a0909a3af